docs/content/en/docs/topics/ingress/traefik/minikube-deployment/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Minikube Deployment"
+linkTitle: "Minikube Deployment"
+weight: 1
+description: |
+  Deploy a local Minikube cluster with Traefik installed and configured.
+---
+
+Minikube is a popular tool for testing k8s clusters locally. When deploying to Minikube, we use the native Kubernetes `port-forward` command from `kubectl` in order to access the services. 
+
+### 1. Install Minikube and start the cluster
+
+Installation instructions for Minikube are [here](https://minikube.sigs.k8s.io/docs/start/).
+
+Once installed, running `minikube start` will bring up a local cluster.
+
+### 2. Install Traefik via Helm
+
+Traefik can be installed via in conjunction with the below Traefik values file - 
```

Comment:
Small typo here, I think you're just missing the word "Helm" in `installed via Helm`

Reply:
Thanks, you're right. I have updated.